src/System.Console/src/System.Console.csproj

```diff
@@ -33,13 +33,12 @@
       <Link>Common\System\Text\ConsoleEncoding.cs</Link>
     </Compile>
   </ItemGroup>
-  <!-- Windows : UAP 
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true' And '$(TargetGroup)' == 'uap101aot' ">
+  <!-- Windows : UAP -->
+  <ItemGroup Condition="'$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uapaot'">
```

Comment:
This does not need the uapaot condition because of there is no uapaot build configuration in https://github.com/dotnet/corefx/blob/master/src/System.Console/src/Configurations.props